tools/ci-scripts/linux/image-cleanup/collector/live_images.py

```diff
@@ -110,8 +110,9 @@ def _images_from_blobs(
         matches = re.finditer(b'^.+CI_BUILD_IMAGE=(.+)\\s*$',
                               git_show_output,
                               re.MULTILINE)
-        image_names.update(map((lambda match: ImageName(str(match.group(1), 'utf-8'))),
-                               matches))
+        matched_names = (ImageName(str(match.group(1), 'utf-8'))
```

Comment:
This is actually the first time I've seen a tuple comprehension! I think a list is a tiny bit more readable (`[ImageName...`), but that may only be because this syntax is new to me. Defer to you.

Reply:
This is actually a [generator expression](https://docs.python.org/3/reference/expressions.html#generator-expressions). It doesn't create an intermediate list/tuple; instead, it creates a generator that we then just pass to the constructor for the set.

Reply:
<whichever emoji best communicates "I have learned something today">